util/dvsim/Deploy.py

```diff
@@ -472,9 +472,12 @@ def __init__(self, index, test, build_job, sim_cfg):
         self.test_obj = test
         self.index = index
         self.build_seed = sim_cfg.build_seed
-        self.seed = RunTest.get_seed()
-        # Systemverilog accepts seeds with a maximum size of 32 bits.
-        self.svseed = int(self.seed) & 0xFFFFFFFF
+
+        base_seed, short_seed, long_seed = RunTest.get_test_seed()
```

Comment:
```suggestion
        self.base_seed, self.short_seed, self.long_seed = RunTest.get_test_seed()
```
And remove the 3 lines below.